packages/notification-center-vue/src/lib/index.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment, @typescript-eslint/naming-convention */
+// @ts-ignore
+import * as Vue from 'vue';
+import {
+  NotificationCenterWebComponent,
+  NotificationCenterComponentProps,
+  NOTIFICATION_CENTER_PROPS,
+} from '@novu/notification-center';
+
+import { defineContainer } from './vue-component-lib/utils';
+
+customElements.define('notification-center-component', NotificationCenterWebComponent);
+
+export const NotificationCenterComponent = defineContainer<NotificationCenterComponentProps>(
+  'notification-center-component',
+  undefined,
+  NOTIFICATION_CENTER_PROPS
+);
```

Comment:
this is basically how to use the Web Component with the Vue... here we do use the `defineContainer` function that was inspired by what the Stencil is doing...
I'll revisit this file to see if we can actually get rid of the `defineContainer`...